dm/syncer/error.go

```diff
@@ -61,6 +65,139 @@ func isDropColumnWithIndexError(err error) bool {
 			strings.Contains(mysqlErr.Message, "with tidb_enable_change_multi_schema is disable"))
 }
 
+// getDDLStatusFromTiDB retrieves the synchronizing status of DDL from TiDB
+// hence here db should be TiDB database.
+func getDDLStatusFromTiDB(ctx context.Context, db *sql.DB, ddl string, createTime int64) (string, error) {
+	rowNum := 10
+	count := 0
+	rowOffset := 0
+
+	showJobsLimit := fmt.Sprintf("ADMIN SHOW DDL JOB QUERIES LIMIT 10 OFFSET %d", rowOffset)
+	rowsLimit, err := db.QueryContext(ctx, showJobsLimit)
+	if err != nil {
+		rowsLimit.Close()
+		return "", err
+	}
+
+	var columnsLimit []string
+	columnsLimit, err = rowsLimit.Columns()
+	if err != nil {
+		rowsLimit.Close()
+		return "", err
+	}
+
+	var resultsLimit [][]string
+	resultsLimit, err = export.GetSpecifiedColumnValuesAndClose(rowsLimit, columnsLimit...)
```

Comment:
`GetSpecifiedColumnValuesAndClose` can extract part of columns from result, so we can pass needed column names as arguments. Please check it's usage in TiDB project.